apps/palette-tauri/src/fonts.css

```
@@ -0,0 +1,36 @@
+/* Self-hosted variable fonts (bundled by Vite into /assets, served from the
+   app's own origin so they satisfy the `font-src 'self'` CSP). These realize the
+   Aurora type stack the tokens already name — without them the palette silently
+   falls back to the OS UI font (Segoe UI on Windows). */
+
+@font-face {
+  font-family: "Manrope";
```

Stylelint (17.12.0)

[error] Expected no quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)